src/main/java/codechicken/nei/recipe/GuiRecipe.java

```diff
@@ -543,20 +543,23 @@ public int openTargetRecipe(BookmarkRecipeId recipeId) {
     }
 
     public List<PositionedStack> getFocusedRecipeIngredients() {
+        List<Integer> indices = getRecipeIndices();
 
-        for (int recipeIndex : getRecipeIndices()) {
-            if (recipeInFocus(recipeIndex)) {
-                return handler.original.getIngredientStacks(recipeIndex);
+        for (int refIndex = 0; refIndex < indices.size(); refIndex++) {
+            if (recipeInFocus(refIndex, indices.get(refIndex))) {
+                return handler.original.getIngredientStacks(indices.get(refIndex));
```

Comment:
Save recipe index into variable (like it's done below) instead of calling method twice.

Reply:
fix